addons/web/static/src/views/view_compiler.js

```diff
@@ -393,7 +404,7 @@ export class ViewCompiler {
      */
     compileGenericNode(el, params) {
         const compiled = createElement(el.nodeName.toLowerCase());
-        const metaAttrs = ["column_invisible", "invisible", "readonly", "required"];
+        const metaAttrs = ["column_invisible", "invisible", "readonly", "required", "technical-chm"];
```

Comment:
doesn't work? Would simply put `technical-chm="1"`, but what we need is a t-if, right?